pages/activities.jsx

```diff
@@ -154,6 +154,15 @@ var ActivitiesPage = React.createClass({
               description="These six hands-on activities engage learners in thinking critically about online privacy by creating secure passwords, understanding how and where their data is being collected, and more. This is an ideal kit for Mozilla Clubs."
             />
             <ActivityKit
+              src1x="https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png"
+              src2x="https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png"
+              title="Privacy Basics: Passwords, Tracking, and Data Retention"
+              level="Intermediate"
+              link="/activities/privacy-basics/"
+              developedBy="Stacy Martin, Senior Data Privacy Manager at Mozilla and the Mozilla Learning Network team"
```

Comment:
given what "erase all kittesn" does, I'm swapping that `,` for a ` - ` instead, and then it won't inject the "and" bit